frontend/src/pages/ConfigurationPage.tsx

```diff
@@ -142,6 +153,21 @@ const RoadmapConfigurationPageComponent = ({
           </button>
         </span>
       </div>
+      {getType(userInfo, currentRoadmap?.id) === RoleType.Admin && (
```

Comment:
The getType result should be stored in a variable, as it is currently computed twice in the component.
```suggestion
      {type === RoleType.Admin && (
```